guides/source/association_basics.md

```diff
@@ -5,18 +5,22 @@ Active Record Associations
 
 This guide covers the association features of Active Record.
 
-After reading this guide, you will know:
+After reading this guide, you will know how to:
 
-* How to declare associations between Active Record models.
-* How to understand the various types of Active Record associations.
-* How to use the methods added to your models by creating associations.
+* Declare associations between Active Record models.
+* Understand the various types of Active Record associations.
+* Use the methods added to your models by creating associations.
```

Comment:
I'm not sure about this change. Many of the other guides have the old form:

https://github.com/rails/rails/blob/016b796ee922567532e7a3bd21160aee7789d448/guides/source/active_record_basics.md?plain=1#L8-L16

https://github.com/rails/rails/blob/016b796ee922567532e7a3bd21160aee7789d448/guides/source/active_support_instrumentation.md?plain=1#L10-L15

https://github.com/rails/rails/blob/016b796ee922567532e7a3bd21160aee7789d448/guides/source/active_record_migrations.md?plain=1#L10-L15

https://github.com/rails/rails/blob/016b796ee922567532e7a3bd21160aee7789d448/guides/source/active_storage_overview.md?plain=1#L8-L14

Reply:
@p8 I was going to follow up and get the rest in a single PR, it seems fine they aren't all exactly the same style intro for one commit?

Reply:
@zzak Sounds good to me.